spec/omniauth/auth0/jwt_validator_spec.rb

```diff
@@ -119,6 +121,36 @@
     end
   end
 
+  describe 'JWT verifier custom issuer' do
+    context 'same as domain' do
+      let(:jwt_validator) do
+        make_jwt_validator(opt_issuer: issuer)
```

Comment:
I rather you pass the issuer value here instead of declaring a global variable that is only going to be used for this single test context.